public/locales/en/translation.json

```diff
@@ -1,4 +1,8 @@
 {
+  "helpNavbar": {
+    "title": "Immediate helpline:",
+    "phoneNum": "+1-340-5558-235"
```

Comment:
if this phone nums not being translated, take it out of here and move it to the code, saying "HELP_PHONE_NUMBER" as a const

notice how if i have to change the help phone number im changing it in two places.